src/pages/ReimbursementAccount/CompanyStep.js

```diff
@@ -1,19 +1,108 @@
+import _ from 'underscore';
 import React from 'react';
-import {View} from 'react-native';
+import {View, ScrollView} from 'react-native';
 import HeaderWithCloseButton from '../../components/HeaderWithCloseButton';
 import CONST from '../../CONST';
 import {goToWithdrawalAccountSetupStep} from '../../libs/actions/BankAccounts';
 import Navigation from '../../libs/Navigation/Navigation';
+import Text from '../../components/Text';
+import TextInputWithLabel from '../../components/TextInputWithLabel';
+import styles from '../../styles/styles';
+import Button from '../../components/Button';
+import FixedFooter from '../../components/FixedFooter';
+import CheckboxWithLabel from '../../components/CheckboxWithLabel';
+import TextLink from '../../components/TextLink';
+import Picker from '../../components/Picker';
+import StatePicker from '../../components/StatePicker';
+import withLocalize, {withLocalizePropTypes} from '../../components/withLocalize';
 
-const CompanyStep = () => (
-    <View>
+const CompanyStep = ({translate}) => (
+    <ScrollView style={[styles.flex1, styles.w100]}>
```

Comment:
Should the Header be inside the `ScrollView`? This way we can always access it? Probably the same would be true for our "Save & Continue" button at the bottom.

e.g. see the behavior of this compared to `/settings/profile`

https://user-images.githubusercontent.com/32969087/122284889-c3750680-ce89-11eb-949b-efa4398bc1d5.mp4

https://user-images.githubusercontent.com/32969087/122284899-c66ff700-ce89-11eb-974c-4fb5c7dea0bf.mp4

Reply:
Good point, made this change 👍